### PR TITLE
fix the bug about adding table

### PR DESCRIPTION
**Pull request by leoppro**
### What problem does this PR solve? 
fix the bug about adding table. this bug is was introduced by #771 .

add some fail-fast check

### Check List 

Tests 

 - Unit test
 - Integration test

### CODE REVIEW

File: cdc/processor.go [hunk 1]
```diff
@@ -508,6 +508,10 @@ func (p *processor) updateInfo(ctx context.Context) error {
 			if err != nil {
 				return false, backoff.Permanent(errors.Trace(err))
 			}
+			err = updatePosition()
+			if err != nil {
+				return false, errors.Trace(err)
```
Comment (leoppro) [inline cdc/processor.go:513]:
```suggestion
				return true, errors.Trace(err)
```

File: cdc/processor.go [hunk 2]
```diff
@@ -806,6 +810,14 @@ func createSchemaStorage(pdEndpoints []string, credential *security.Credential,
...
 
 func (p *processor) addTable(ctx context.Context, tableID int64, replicaInfo *model.TableReplicaInfo) {
+	globalResolvedTs := atomic.LoadUint64(&p.sinkEmittedResolvedTs)
+	/*
+		if replicaInfo.StartTs < globalResolvedTs {
+			log.Fatal("cannot add table which of startTs is less than the global resolved, please report a bug",
+				zap.Int64("tableID", tableID), zap.Any("replicaInfo", replicaInfo),
+				zap.Uint64("globalResolvedTs", globalResolvedTs))
+		}
+	*/
```
Comment (overvenus) [inline cdc/processor.go:820]: How about removing '/*' and '*/'?

Comment (leoppro) [reply cdc/processor.go:820]: There's a bug that's causing this place to fatal. I'm working on it.

Comment (leoppro) [reply cdc/processor.go:820]: I will fix it in next PR

**Pull request merged**